server-preload.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Use node.js preloading to require a node module before importing and executing
+ * the entrypoint script.
+ *
+ * Intended to be used in package.json scripts with something like:
+ * - `node -r ./server-preload.js ./node_modules/.bin/next start -p 8080`
+ *
+ * We need to connect to App Insights early enough for it to be able to log HTTP requests.
+ *
+ * References:
+ * - https://jake.tl/notes/2021-04-04-nextjs-preload-hack
+ * - https://nodejs.org/api/cli.html#cli_r_require_module
+ */
+
+function setupLogging() {
+  // Connect to application insights during preload to capture HTTP requests.
+  const appInsights = require('applicationinsights')
+  appInsights
+    .setup(process.env.APPLICATIONINSIGHTS_CONNECTION_STRING)
+    .setAutoCollectRequests(true) // Default setting, but explicitly set to true for clarity
+    .setSendLiveMetrics(true) // Send Live Metrics to App Insights
+    .setAutoCollectConsole(true, true) // Pass the second argument to also log console.log() statements
```

Comment:
I'm not 💯 on whether or not we want to also log Next.js's `console.log()` statements. It seemed useful (you can see them in `traces`, but we can easily turn this off.

Reply:
I think we shouldn't/can't because we'll end up logging claims data (which is not tied to any identifier, so perhaps not an actual issue, but potentially a perceived one)

Reply:
I think the intention is to disable all `console.log` and `console.dir` calls, so I don't think that should be an issue.

Reply:
That's true, but I'm thinking that if we really want to log something, it's not too difficult for us to use the logger instead of console.log, whereas there is a nonzero chance that we will accidentally end up deploying a console.log to production at some point, because we are human :P

Reply:
Let's talk more about this synchronously. `console.log()` logs still get logged to Azure.